index.html

```diff
@@ -308,20 +308,153 @@
 
     <section class="informative">
       <h2>Introduction</h2>
+
+      <p>
+Digital signatures, based on
+<a href="https://en.wikipedia.org/wiki/Public-key_cryptography">asymmetric
+cryptography</a>, can be used in [=authentication=] and [=authorization=]
+schemes to make them difficult for adversaries to compromise. However,
+one shortcoming of digital signatures is the challenge in disseminating
+necessary information, such as public cryptographic keys, to those who need
+to verify the security of a digital signature.
```

Comment:
Hey @jandrieu can you pull this change set into #102, please? @dlongley had some good language here that we don't want to lose.